doc/devel/documenting_mpl.rst

```diff
@@ -139,6 +139,54 @@ An example docstring looks like:
 The Sphinx website also contains plenty of documentation_ concerning ReST
 markup and working with Sphinx in general.
 
+.. note::
+
+   Some parts of the documentation do not yet conform to the current
+   documentation style. If in doubt, follow the rules given here and not what
+   you may see in the source code. Pull requests updating docstrings to
+   the current style are very welcome.
+
+Additional formatting conventions
+---------------------------------
+
```

Comment:
This all looks good to me.  Note I knocked all this down a level in the outline, so we'll have to remember to reconcile.  Actually my new outlining is somewhat incomplete the way I've done it, but I thought I'd leave it like that to at least remind myself that it needed fixing.